main.go

```diff
@@ -216,7 +221,13 @@ func runLoop(ctx context.Context) {
 
 		if config.OSInventoryEnabled() {
 			// This should always run after ospackage.SetConfig.
-			inventory.Run()
+			client, err := agentendpoint.NewClient(ctx)
+			if err != nil {
+				logger.Fatalf(err.Error())
```

Comment:
Don't log fatal here, just error. Actually for this implementation you should create the client inside tasker.Enqueue (dont need to change the "inventory" case above as that is only for the command line)

Reply:
Updated. Anything special happens with fatal logging?

Reply:
Fatal is the same as Error but with and os.Exit(1)  :)

Reply:
I see thanks